business/core/home/event.go

```diff
@@ -0,0 +1 @@
+package home
```

Comment:
I am wondering if we should have this file if there are no events?

Reply:
I created it and but I'm not sure how events work, so just left it empty, I'll delete it for the next PR

Reply:
Nah, leave the file for now. I left this comment to start a conversation. I think it's good to have it.